DependencyInjection/MonologExtension.php

```diff
@@ -291,6 +291,43 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 $handler['bubble'],
             ));
             break;
+        case 'redis':
+        case 'predis':
```

Comment:
I think this does not work, since you are using `$handler['redis]` bellow (and not `$handler['predis]`)